bin/i18n/resources/dashboard/curriculum_content/sync_down.rb

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../i18n_script_utils'
+require_relative '../../../utils/sync_down_base'
+require_relative '../curriculum_content'
+
+module I18n
+  module Resources
+    module Dashboard
+      module CurriculumContent
+        class SyncDown < I18n::Utils::SyncDownBase
+          config.download_paths << DownloadPath.new(crowdin_src: DIR_NAME)
```

Comment:
Sometimes we use `FILE_PATH`  and sometimes `DIR_NAME`. Some i18n resources are just a single file, while other are spread through multiple files (such as in course and curriculum content) therefor we need to proved the directory to crowdin.